railties/lib/rails/api/task.rb

```diff
@@ -92,19 +96,14 @@ def desc(description)
       end
 
       def load_and_configure_sdoc
-        require "sdoc"
```

Comment:
This require was originally here to handle a possible missing requirement on SDoc in applications' `Gemfile`, but since as of https://github.com/rails/rails/commit/cd7cc5254b090ccbb84dcee4408a5acede25ef2a this is no longer possible to generate the API from an application, we should rather rename the method `configure_sdoc` and require our custom generator anyway. What do you think ?